dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -57,9 +47,33 @@ if [[ ! -z "$NAMESPACE_ID" ]]; then
 fi
 
 if [ -z "$CONFIG_TYPE" ] || [ "$CONFIG_TYPE" == "separated" ]; then
+    CFGGEN_PARAMS=" \
+        -d \
+        -y /etc/sonic/constants.yml \
+        -t /usr/share/sonic/templates/bgpd/gen_bgpd.conf.j2,/etc/frr/bgpd.conf \
+        -t /usr/share/sonic/templates/zebra/zebra.conf.j2,/etc/frr/zebra.conf \
+        -t /usr/share/sonic/templates/staticd/gen_staticd.conf.j2,/etc/frr/staticd.conf \
+    "
+    MGMT_FRAMEWORK_CONFIG=$(echo $FRR_VARS | jq -r '.frr_mgmt_framework_config')
+    if [ -n "$MGMT_FRAMEWORK_CONFIG" ] && [ "$MGMT_FRAMEWORK_CONFIG" != "false" ]; then
+        CFGGEN_PARAMS="$CFGGEN_PARAMS \
+            -t /usr/local/sonic/frrcfgd/bfdd.conf.j2,/etc/frr/bfdd.conf \
+            -t /usr/local/sonic/frrcfgd/ospfd.conf.j2,/etc/frr/ospfd.conf \
+        "
+    fi
```

Comment:
The original logic is to remove the files otherwise. Wondering if the removal is mandatory? For example, if the files exist previously, is it necessary to remove them here even if it is not generated again from the template?

Reply:
Yes, good question. I'm not sure whether any effect will involve if the previous existing files are kept in the `/etc/frr/` folder to the `unified` mode. So I just want to keep the original behavior.

Reply:
I think this is somewhat different from the original behavior though. Let's assume this scenario:
1. `CONFIG_TYPE` is `separated` and `$MGMT_FRAMEWORK_CONFIG` is `false`
2. there are `bfdd.conf` and `ospfd.conf` in `etc/frr` before

The original behavior is to remove `bfdd.conf` and `ospfd.conf`;
The new behavior is to keep `bfdd.conf` and `ospfd.conf`

Reply:
Got it, you are right.
I try to remove it in the `else` block. Please review it.